### PR TITLE
DOC-4314 added note about auto flush after disconnection

### PR DESCRIPTION
**Pull request by andy-stark-redis**
[DOC-4314](https://redislabs.atlassian.net/browse/DOC-4314)

[DOC-4314]: https://redislabs.atlassian.net/browse/DOC-4314?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/develop/connect/clients/client-side-caching.md [hunk 1]
```diff
@@ -63,6 +61,12 @@ will evict the stale data in response. Next time a client reads from
 the same key, it will access the database directly and refresh its cache
 with the updated data.
 
+{{< note >}}If any connection from a client gets disconnected (including
+one from a connection pool) then the client will flush all keys from the
```
Comment (kaitlynmichael) [inline content/develop/connect/clients/client-side-caching.md:65]:
```suggestion
one from a connection pool), then the client will flush all keys from the
```

File: content/develop/connect/clients/python/redis-py.md [hunk 2]
```diff
@@ -228,6 +228,9 @@ r.hget("person:2", "name") # Read from the server
 r.hget("person:2", "name") # Read from the cache
 ```
 
+Note that the client will also flush the cache automatically
```
Comment (kaitlynmichael) [inline content/develop/connect/clients/python/redis-py.md:231]:
```suggestion
The client will also flush the cache automatically
```

File: content/develop/connect/clients/java/jedis.md [hunk 3]
```diff
@@ -309,6 +309,10 @@ client.hget("person:2", "name");    // Read from the server
 client.hget("person:2", "name");    // Read from the cache
 ```
 
+Note that the client will also flush the cache automatically
```
Comment (kaitlynmichael) [inline content/develop/connect/clients/java/jedis.md:312]:
```suggestion
The client will also flush the cache automatically
```

**Review by kaitlynmichael — APPROVED** [review]
A few small suggestions. Looks good!

**Comment by andy-stark-redis** [comment]
Suggestions applied. Thanks @kaitlynmichael !

**Pull request merged**